<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseManager</class>
 <widget class="QWidget" name="BrowseManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="laIP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>48</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>IP地址：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbAliasName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>选择一个目录</string>
   </property>
   <item>
    <property name="text">
     <string>选择目录</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pbBrowse">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>打开并浏览文件</string>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCopy">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>将链接复制到剪贴板</string>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbIP">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <zorder>pbCopy</zorder>
  <zorder>laIP</zorder>
  <zorder>cbAliasName</zorder>
  <zorder>pbBrowse</zorder>
  <zorder>cbIP</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
